vrk-kpa/suomidigi
# SUOPA-450 Comments

To test: 
- `make fresh`
- In the UI `flush all caches`
- Log in with couple of different users (f.e. admin and yourself)
   - Edit your own account and add First name and Last name (and there's no picture, add a picture)
- Go to some article or core content and see if there's a comment form at the bottom of the page.
   - If so, Add a few with different users and try to edit them also. 
   - Also check that the comments section looks ok when you're not logged in
